boards/airfy-beacon/include/periph_conf.h

```diff
@@ -98,16 +98,9 @@ static const timer_conf_t timer_config[] = {
  * @name ADC configuration
  * @{
  */
-#define ADC_NUMOF           (1U)
-#define ADC_0_EN            1
-#define ADC_MAX_CHANNELS    4
-
-/* ADC 0 device configuration */
-#define ADC_0_CHANNELS      4
-#define ADC_0_CH0           ADC_CONFIG_PSEL_AnalogInput3
-#define ADC_0_CH1           ADC_CONFIG_PSEL_AnalogInput4
-#define ADC_0_CH2           ADC_CONFIG_PSEL_AnalogInput5
-#define ADC_0_CH3           ADC_CONFIG_PSEL_AnalogInput6
+static const uint8_t adc_config[] = {3, 4, 5, 6};
```

Comment:
If its included only once, we would be fine, but since it this is the periph_conf this probably is included multiple times => every include to a new compile unit potentially generates a new array (but maybe is optimized out?). Also applicable for below.